src/app-init/router.js

```diff
@@ -33,6 +34,7 @@ routerConfig(
       { name: ROUTE_WIDGET, path: '/widget/view/:widget' },
       { name: ROUTE_FRAGMENT_ADD, path: '/fragment/add' },
       { name: ROUTE_FRAGMENT_EDIT, path: '/fragment/edit/:fragmentCode' },
+      { name: ROUTE_PAGE_SETTINGS, path: '/pageSettings' },
```

Comment:
It would be good to have the path = `/page/settings` for coherence. Also if possible please place it along the other ROUTE_PAGE_* routes